launch/fake_driver.launch

```diff
@@ -1,9 +1,7 @@
 <?xml version="1.0" ?>
 <launch>
-    <param name="robot_description" command="$(find xacro)/xacro --inorder $(find robotont_description)/urdf/robotont_base.urdf.xacro"/>
-    
-    <node name="joint_state_publisher" pkg="joint_state_publisher" type="joint_state_publisher" />
-    <node name="robot_state_publisher" pkg="robot_state_publisher" type="robot_state_publisher" />
-    <node name="fake_driver_node" pkg="robotont_driver" type="fake_driver_node" output="screen"/>    
-	<node name="rviz" pkg="rviz" type="rviz" args="-d $(find robotont_driver)/config/robotont_odom.rviz"/>
+  <include file="$(find robotont_description)/launch/display_simulated_robot.launch">
+    <arg name="fixed_frame" value="odom" />
```

Comment:
see `<arg>` teeb katki. launch faili ei toimi. piisab, kui oleks ainult:
`<include file="$(find robotont_description)/launch/display_simulated_robot.launch" />`

Reply:
testid ikka melodic-devel-veiko haruga robotont_description kimbus? mõlemal pool on muutusi

Reply:
aga muidugi ei testi

Reply:
testimine käib ju alati olemasolevate kinnitatud asjade vastu. muidu lähevad asjad katki.

Reply:
Aga kuidas siis muna-kana muutused sisse viia?

Reply:
nüüd ma näen jah, et seal harus on asi juba fixitud, aga saada siis too muudatus laivi. muidu peame ka broke-the-repo wall-of-shame'i tegema ;)

Reply:
https://github.com/robotont/robotont_description/pull/5

Reply:
no antud juhul oli sul tegelikult descriptionisse viidud sisse muudatus, mis midagi ära ei lõhkunud, seega oleks descriptioni muudatus olnud kindel valik kõigepealt sisse viia. see poleks tavakasutaja jaoks midagi muutnud. seega väga hästi implementitud,